src/component/mxgraph/shape/activity-shapes.ts

```diff
@@ -44,7 +44,7 @@ export abstract class BaseActivityShape extends mxRectangleShape {
 
   public paintForeground(c: mxAbstractCanvas2D, x: number, y: number, w: number, h: number): void {
     super.paintForeground(c, x, y, w, h);
-    this.paintMarkerIcons(buildPaintParameter(c, x, y, w, h, this, 0.17, false, 1.5));
+    this.paintMarkerIcons(buildPaintParameter(c, x, y, w, h, this, 0, false, 1.5));
```

Comment:
:information_source: I suggest to put `undefined` instead of `0` for the ratio as the TSDoc explain to do this to keep the original icon size
Anyway, this should change in the future with #211

Reply:
this is little tricky as if we pass undefined the default value (0.25) is used, we should discuss it as it requires further investigation

Reply:
Yes you are righ

I was reminding me using `undefined` but this was by set the `ratioFromParent` property on a `PaintParameter` instance built by the `buildPaintParameter` function :stuck_out_tongue_winking_eye:.
For now, we can just add a comment to explain the `0` value or introduce a variable/constant with an explicit name to hold the meaning of this value